src/components/IOUConfirmationList.js

```diff
@@ -36,6 +36,10 @@ const propTypes = {
     // Callback to update comment from IOUModal
     onUpdateComment: PropTypes.func,
 
+    /** Callback to update payment type in IOUModal */
+    /* eslint-disable-next-line react/no-unused-prop-types */
+    onUpdatePaymentType: PropTypes.func.isRequired,
+
```

Comment:
I had to add `onUpdatePaymentType` also to `IOUConfirmationList` so we can pass the `paymentType` along all the way to `IOUModal` component.